docs/rackspace-elastic-engineering-and-optimizer/index.rst

```diff
@@ -42,47 +42,48 @@ between Rackspace Technology and the customer.
-   * - DevOps and Automation
-     - * Manage, maintain, and enhance automation systems for operational effectiveness, stability and security
-       * Change existing automation systems, or develop new automation systems
-   * - Performance Management
+   * - DevOps and automation
+     - * Manage, maintain, and enhance automation systems for operational effectiveness, stability, and security
+       * Change existing automation systems or develop new automation systems
+   * - Performance management
      - * Identify performance targets and continuously adjust infrastructure to deliver the desired performance at the optimal cost
-       * Tune applications, provide detailed analysis of application-level performance problems
-   * - Database Management
-     - * Scale databases, provide performance management and maintenance services
+       * Tune applications and provide detailed analysis of application-level performance problems
+   * - Database management
+     - * Scale databases and provide performance management and maintenance services
        * Create database clusters, read slaves, and so on
-   * - Application Operations
+   * - Application operations
      - * Manage the configuration and patching of common, off-the-shelf applications
        * Provide custom upgrades and patches for off-the-shelf applications, develop configuration frameworks and methodologies for custom applications
-   * - Security Management
+   * - Security management
      - * Use architectural best practices to maintain environment security
        * Respond to security incidents identified by the Customer Security Team
        * Deploy customer-provided third-party security tools
        * Create reports or contribute to reports that a customer needs as part of an infrastructure audit for certification purposes
-   * - Disaster Recovery Planning
+   * - Disaster recovery planning
      - * Run disaster recovery solutions, extend existing solutions, and run customer-directed tests of disaster recovery systems
-   * - Planned Maintenance
+   * - Planned maintenance
      - * Perform preventative maintenance so that infrastructure, database, and application operations are stable and secure
-       * Perform optional maintenance and customer-directed changes, which can include applying security patches on a regular basis
-   * - Cost Optimization
+       * Perform optional maintenance and customer-directed changes, which can include applying security patches regularly
+   * - Cost optimization
      - * Implement architectural changes to support cost management decisions
-   * - Advice and Support
-     - * Answer questions related to cloud usage, best practices, and make recommendations
+   * - Advice and support
+     - * Answer questions and make recommendations related to cloud usage and best practices
+
 
 .. _monitoring_resolution:
 
-Advanced Monitoring and Resolution
+Advanced monitoring and resolution
```

Comment:
Is this a product/service offering name, and should it retain capitalization?

Reply:
it is a product